pages/common/tail.md

```diff
@@ -5,24 +5,24 @@
 
 - Show last 'num' lines in file:
 
-`tail -n {{num}} {{file}}`
+`tail --lines {{count}} {{path/to/file}}`
 
 - Show all file since line 'num':
 
-`tail -n +{{num}} {{file}}`
+`tail --lines +{{count}} {{path/to/file}}`
 
 - Show last 'num' bytes in file:
 
-`tail -c {{num}} {{file}}`
+`tail --bytes {{count}} {{path/to/file}}`
 
 - Keep reading file until `Ctrl + C`:
 
-`tail -f {{file}}`
+`tail --follow {{path/to/file}}`
 
 - Keep reading file until `Ctrl + C`, even if the file is inaccessible:
 
-`tail -F {{file}}`
+`tail --follow={{path/to/file}} --retry`
```

Comment:
This "sounds" better, also the `=` makes some dark magic that I don't understand yet.
```suggestion
`tail --retry --follow {{path/to/file}}`
```
I don't know what this means, the `all.json` exists:
```bash
$ tail --retry --follow=all.json
tail: invalid argument ‘all.json’ for ‘--follow’
Valid arguments are:
  - ‘descriptor’
  - ‘name’
```
The command with `=` doesn't fail if the file is not created yet, and follow the file when it appears.